packages/actions-global/src/actions/handle-action-error.ts

```diff
@@ -1,4 +1,4 @@
-import { Dispatch } from 'redux';
+import { ThunkDispatch } from 'redux-thunk';
```

Comment:
I'm not sure why you would need this, because we're dispatching "normal actions" and we just need the normal `dispatch` function.

The thunk that you maybe refer to is `handleActionError`, but the type here refers to the `dispatch` function used by the thunk.

Reply:
@emmenko Can you please take over this PR? Because, looks like you have a certain idea in mind of how that should be implemented. So, probably will be just faster for you to do that. Thanks! ❤️

Reply:
Just revert this import, the rest is fine 😄

Reply:
That will cause type check errors. I fixed that type for a reason.

Reply:
Ah because in the hook you now call the action like `dispatch(handleActionError(error))` instead of `handleActionError(error)(dispatch)`.

Ok sorry, I missed that.

We need to include the `redux-thunk` as a dependency though.

Reply:
https://github.com/commercetools/merchant-center-application-kit/pull/1335/commits/99d69fef251afbd8e8673c67548af1529364e2fb